cloud/passportauth/src/auth/PassportAuth.ts

```diff
@@ -46,10 +46,12 @@ export interface Auth {
  */
 export class PassportAuth implements Auth {
   protected loginRoute: string;
-  private log: Logger = new BunyanLogger({ name: 'Passport-Auth', level: 'error' });
+  protected loginErrorRoute: string;
+  private log: Logger = new BunyanLogger({name: 'Passport-Auth', level: 'error'});
 
-  constructor(protected readonly userSec: UserSecurityService, loginRoute?: string) {
+  constructor(protected readonly userSec: UserSecurityService, loginRoute?: string, loginErrorRoute?: string) {
```

Comment:
We do not need that here. Login route is used to mount endpoints and mark passport. For error route we should pass that directly to function